README.md

```diff
@@ -12,11 +12,64 @@ Installation of the [npm package](https://npmjs.org/package/xo-server-transport-
 
 ## Usage
 
-**TODO**
+Like all other xo-server plugins, it can be configured directly via
+the web iterface, see [the plugin documentation](https://xen-orchestra.com/docs/plugins.html). You can also test the configuration plugin if it works.
+
+### Slack
+
+#### Generate the Webhook
+
+1. Log in your Slack team account
+2. Click on the **Main menu** at the top and choose **Apps & Integrations**
+
+   ![Apps & Integrations](image/DocImg1.png)
+3. Search **Incoming WebHooks**
```

Comment:
No, between text ("3. Search...") and image (DocImg2.png). Otherwise, the image is on the same line as the text.